client/package.json

```diff
@@ -43,6 +48,8 @@
     ]
   },
   "devDependencies": {
-    "prettier": "2.4.1"
+    "moment": "^2.29.1",
+    "prettier": "2.4.1",
+    "yup": "^0.32.11"
```

Comment:
I don't remember. At the beginning, I thought that I would use these libraries, but I created custom validation. But in the future, I think I will use moment and yup for validation with the registrations field.

Reply:
Yeah, it's ok to use any library, I was wondering why it's in `"devDependencies"`, and not in `"dependencies"`? 😊 
here is a good explanation of the difference https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies

usually, devDependencies are used to reduce bundle and build time for the production build, but nowadays people almost don't use it, probably because of the more complex CI/CD processes, which usually require prod and dev scripts to run (testing, lining).